src/Illuminate/Database/Console/Migrations/MigrateMakeCommand.php

```diff
@@ -107,11 +95,11 @@ public function handle()
      */
     protected function writeMigration($name, $table, $create)
     {
-        $file = $this->creator->create(
+        return tap($this->creator->create(
             $name, $this->getMigrationPath(), $table, $create
-        );
-
-        $this->components->info(sprintf('Migration [%s] created successfully.', $file));
+        ), function ($file) {
+            $this->components->info(sprintf('Migration [%s] created successfully.', $file));
+        });
```

Comment:
This function is supposed to return a string (per the docblock) and I figure it would be good to expose the file written.